docs/application-connector/docs/010-details-ac-deployment.md

```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
+
+``` yaml
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: production-1
+spec:
+  description: This Remote Environment for connecting production system 1.
+  labels:
+    region: us
+    kind: production
 ```
 
-The **global.domainName** is mandatory. Example values can look like:
+Then you can execute the following command:
+
+``` bash
+kubectl apply -f ./re-production-1.yaml
 ```
-wormhole.cluster.kyma.cx
-nightly.cluster.kyma.cx
+
+## Delete a Remote Environment
+
+The Remote Environment can be removed from Kyma using either Console UI or Kubectl.
+
+### Using the UI:
+
+- Go to the Kyma console UI.
+- Select Administration.
+- Select the Remote Environments from the Integration section.
+- Choose the Remote Environment to which you want to delete.
+- Click Delete.
+
+![Delete RE](./assets/delete-re.png)
+ 
+
+### Using Kubectl
+
+The Remote Environment can be deleted using the following command:
+
+Then you can execute the following command:
+
+```bash
+kubectl delete re name-of-remote-environment
 ```
 
-You can override the following parameters:
+## Update Remote Environment
 
-- **sourceEnvironment** is the Event source Environment name.
-- **sourceType** is the Event source type.
-- **sourceNamespace** is the organization that publishes the Event.
+The Remote Environment can be updated using either Console UI or Kubectl.
 
-## Work with Helm
+>**NOTE:** The name of Remote Environment cannot be changed.
 
-Helm provides the following commands:
-- `helm list` lists existing Helm releases
-- `helm test [release-name]` tests a release
-- `helm get [release-name]` shows the contents of `.yaml` files that make up the release
-- `helm status [release-name]` shows the status of a named release
-- `helm delete [release-name]` deletes a release from Kubernetes
+### Using the UI:
 
-The full list of the Helm commands is available in the [Helm documentation](https://docs.helm.sh/helm/).
-You can also use the `helm --help` command.
+- Go to the Kyma console UI.
+- Select Administration.
+- Select the Remote Environments from the Integration section.
+- Choose the Remote Environment to which you want to update.
+- Change the description and labels
+- Click Save.
+
+### Using Kubectl
+
+Update the file: `re-production-1.yaml`
+
+``` yaml
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: production-1
+spec:
+  description: This is new description.
+  labels:
+    region: new-region
+    kind: production
+```
+
+Then you can execute the following command:
+
+``` bash
+kubectl apply -f ./re-production-1.yaml
+```
 
-## Use kubectl
+## How to check if your remote environment was successfully created.
 
-To check if everything runs correctly, use the `kubectl get pods -n kyma-integration` or `kubectl get services -n kyma-integration` command.  
+The new Remote Environment will appear on the Remote Environment lists and will have status `Serving.`
```

Comment:
``The new Remote Environment appears on the **Remote Environments** list with the `Serving` status.``